kpops/components/streams_bootstrap_v3/streams/model.py

```diff
@@ -0,0 +1,267 @@
+from kpops.components.common.streams_bootstrap import StreamsBootstrapValues
+from kpops.components.common.topic import KafkaTopic, KafkaTopicStr
+from kpops.components.streams_bootstrap_v3.kafka_app import (
+    KafkaAppValues,
+    KafkaStreamsConfig,
+)
+from kpops.utils.docstring import describe_attr
+from kpops.utils.pydantic import (
+    CamelCaseConfigModel,
+    DescConfigModel,
+)
+
+
+class StreamsConfig(KafkaStreamsConfig):
+    """Streams Bootstrap kafka section.
+
+    :param application_id: Unique application ID for Kafka Streams. Required for auto-scaling
+    :param input_topics: Input topics, defaults to []
+    :param input_pattern: Input pattern, defaults to None
+    :param labeled_input_topics: Extra input topics, defaults to {}
+    :param labeled_input_patterns: Extra input patterns, defaults to {}
+    :param error_topic: Error topic, defaults to None
+    :param config: Configuration, defaults to {}
+    :param delete_output: Whether the output topics with their associated schemas and the consumer group should be deleted during the cleanup, defaults to None
+    """
+
+    application_id: str | None = Field(
+        default=None,
+        title="Unique application ID",
+        description=describe_attr("application_id", __doc__),
+    )
+    input_topics: list[KafkaTopicStr] = Field(
+        default=[], description=describe_attr("input_topics", __doc__)
+    )
+    input_pattern: str | None = Field(
+        default=None, description=describe_attr("input_pattern", __doc__)
+    )
+    labeled_input_topics: dict[str, list[KafkaTopicStr]] = Field(
+        default={}, description=describe_attr("extra_input_topics", __doc__)
+    )
+    labeled_input_patterns: dict[str, str] = Field(
+        default={}, description=describe_attr("extra_input_patterns", __doc__)
+    )
+    error_topic: KafkaTopicStr | None = Field(
+        default=None, description=describe_attr("error_topic", __doc__)
+    )
+    config: dict[str, Any] = Field(
+        default={}, description=describe_attr("config", __doc__)
+    )
+    delete_output: bool | None = Field(
+        default=None, description=describe_attr("delete_output", __doc__)
+    )
+
+    @pydantic.field_validator("input_topics", mode="before")
+    @classmethod
+    def deserialize_input_topics(
+        cls, input_topics: list[str] | Any
+    ) -> list[KafkaTopic] | Any:
+        if isinstance(input_topics, list):
+            return [KafkaTopic(name=topic_name) for topic_name in input_topics]
+        return input_topics
+
+    @pydantic.field_validator("labeled_input_topics", mode="before")
+    @classmethod
+    def deserialize_labeled_input_topics(
+        cls, labeled_input_topics: dict[str, str] | Any
+    ) -> dict[str, list[KafkaTopic]] | Any:
+        if isinstance(labeled_input_topics, dict):
+            return {
+                label: [KafkaTopic(name=topic_name) for topic_name in topics]
+                for label, topics in labeled_input_topics.items()
```

Comment:
`topics: str` seems like a mistake

Reply:
most likely I made the mistake in https://github.com/bakdata/kpops/pull/515

Reply:
It was a typo by me. I forgot to update the field name in the `field_validator` decorator